cw_decred/lib/transaction_info.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:cw_core/transaction_direction.dart';
+import 'package:cw_core/keyable.dart';
+import 'package:cw_core/transaction_info.dart';
+import 'package:cw_core/transaction_direction.dart';
+import 'package:cw_core/format_amount.dart';
+import 'package:cw_core/wallet_type.dart';
+import 'package:cw_decred/amount_format.dart';
+
+class DecredTransactionInfo extends TransactionInfo {
+  DecredTransactionInfo({
+    required id,
+    required amount,
+    required fee,
+    required direction,
+    required isPending,
+    required date,
+    required height,
+    required confirmations,
+    required to,
+  }) {
```

Comment:
No type mean dynamic types. Does this compile? Even if it does, it might help readability to add the types:
```suggestion
  DecredTransactionInfo({
    required String id,
    required int amount,
    int? fee,
    required TransactionDirection direction,
    required bool isPending,
    required DateTime date,
    required int height,
    required int confirmations,
    String? to,
  }) {
```

Reply:
The super class has these fields so apparently it uses those. Ill see if adding the type also works.

Reply:
Added the types. Can they just all be required? We should always know the fee and to addr I think.